hooks/update-events.ts

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs'
+import util from 'util'
+
+import { fetchCommunityEvents } from './event-conversion-utils'
+
+export default async function (apiKey, outputFolder: string) {
+  const writeFile = util.promisify(fs.writeFile)
+  const futureCommunityEvents = await fetchCommunityEvents(apiKey, { days: 31 })
+  const pastCommunityEvents = await fetchCommunityEvents(apiKey, { days: -31 })
+  const writeFutureEvents = writeFile(`${outputFolder}/future-community-events.json`, JSON.stringify(futureCommunityEvents, null, 2))
```

Comment:
Could you move the line `const writeFile = util.promisify(fs.writeFile)` right before `const writeFutureEvents...` in order to have the functionality together?